db/migrate/20230929095022_create_round.rb

```diff
@@ -0,0 +1,13 @@
+class CreateRound < ActiveRecord::Migration[7.0]
+  def change
+    create_table :rounds do |t|
```

Comment:
TODO:
Link round to game_player rather than game AND player. Also update the models relation, and the next migrations index